### DISCUSSION TITLE
A question about GNSS satellite boxWing model

### DISCUSSION THREAD

**Discussion by LNTU-YS** [groops-devs/groops#297] [Q&A] [answered]
Hello,

I would like to know more about GNSS satellite boxWing model. May I ask if you can provide the latest GNSS satellite boxWing model information?

Thank you in advance for your feedback and help.
With Kind Regards
YUAN

**Comment by tmayerguerr** [comment, marked as answer]
Hi YUAN,

We use the latest information from [igs_satellite_metadata.snx](https://files.igs.org/pub/station/general/igs_satellite_metadata.snx) and converted it with [SinexMetadata2SatelliteModel](https://groops-devs.github.io/groops/html/SinexMetadata2SatelliteModel.html) into GROOPS format using `{groopsDataDir}/gnss/transmitter/satelliteModel/satelliteModel_boxWing.{block}.xml` as inputfileSatelliteModel.

best regards
Torsten

**Reply by LNTU-YS** [reply]
Hi Torsten,

Thank you very much for your guidance.

With Kind Regards
YUAN

**Reply by GNSSG** [reply]
同学你好，请问方便留一个联系方式嘛？关于groops我还有一些问题想跟你讨论。

**Reply by LNTU-YS** [reply]
你好，我不清楚能不能私信 ？
而且我也是新手上路，知道的有限。
